apps/tailwind-components/components/input/Listbox.vue

```diff
@@ -117,7 +117,7 @@ const searchElemRef = ref<InstanceType<typeof InputSearch>>();
 const displayText = ref<string>(props.placeholder);
 const startingCounter = ref<number>(0);
 const selectedElementId = ref<string>("");
-const searchTerm = defineModel<string>("");
+const searchTerm = defineModel<string>("search");
```

Comment:
I think the initial value should be empty. Otherwise, it will be included in the query

Reply:
this is not the initial value, but the name of the model.

Reply:
(we now have the model and a model:search)